Roles/Neutral/Taskinator.cs

```diff
@@ -0,0 +1,139 @@
+﻿using Hazel;
+using System;
+using System.Collections.Generic;
+using static TOHE.Translator;
+
+namespace TOHE.Roles.Neutral;
+
+public static class Taskinator
+{
+    private static readonly int Id = 8670;
+    private static List<byte> playerIdList = new();
+    public static bool IsEnable = false;
+
+    public static Dictionary<byte, List<int>> taskIndex = new();
+    public static Dictionary<byte, int> TaskMarkPerRound = new();
+    private static int maxTasksMarkedPerRound = new();
+
+    public static OptionItem TaskMarkPerRoundOpt;
+
+    public static void SetupCustomOption()
+    {
+        Options.SetupRoleOptions(Id, TabGroup.NeutralRoles, CustomRoles.Taskinator);
+        TaskMarkPerRoundOpt = IntegerOptionItem.Create(Id + 10, "TaskMarkPerRound", new(1, 14, 1), 3, TabGroup.NeutralRoles, false).SetParent(Options.CustomRoleSpawnChances[CustomRoles.Taskinator])
+            .SetValueFormat(OptionFormat.Votes);
+        Options.OverrideTasksData.Create(Id + 11, TabGroup.CrewmateRoles, CustomRoles.Taskinator);
+    }
+
+    public static void Init()
+    {
+        playerIdList = new();
+        taskIndex = new();
+        TaskMarkPerRound = new();
+        IsEnable = false;
+        maxTasksMarkedPerRound = TaskMarkPerRoundOpt.GetInt();
+    }
+    public static void Add(byte playerId)
+    {
+        playerIdList.Add(playerId);
+        TaskMarkPerRound[playerId] = 0;
+        IsEnable = true;
```

Comment:
Seems `playerIdList` it's not used anywhere, so it can be removed

Reply:
I plan on using it in the future, for a different role

Reply:
Hmm, ok